workers/tasks/export/html.tsx

```diff
@@ -176,20 +190,28 @@ const renderFrontMatter = ({
 	updatedDateString,
 	publishedDateString,
 	primaryCollectionTitle,
+	primaryCollectionMetadata,
 	doi,
 	title,
 	communityTitle,
 	accentColor,
 	attributions,
 	licenseSlug,
+	copyrightYear,
 	publisher,
 }: PubMetadata) => {
 	const affiliations = [
 		...new Set(attributions.map((attr) => attr.affiliation).filter((x) => x)),
 	];
 	// do not put community title if this is a book
+	const pubPublisher =
```

Comment:
This is another thing that feels like it should go into a utility function somewhere, if only so we're forced to name it.